src/Sdk.php

```diff
@@ -1,100 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Sentry;
-
-use Sentry\State\Hub;
-
-/**
- * Creates a new Client and Hub which will be set as current.
- *
- * @param array $options The client options
- */
-function init(array $options = []): void
```

Comment:
This removal is a huge BC.

Reply:
I fu**ed up the rebase... Thanks for spotting this!

Reply:
I fu**ed up the rebase, this file has only been renamed and nothing got deleted